src/libutil/util.hh

```diff
@@ -120,7 +120,7 @@ std::optional<N> string2Int(const std::string_view s)
 template<class N>
 N string2IntWithUnitPrefix(std::string_view s)
 {
-    N multiplier = 1;
+    uint64_t multiplier = 1;
```

Comment:
intentional?

Reply:
oh so the prefix doesn't overflow?

Reply:
Yes, this is needed when `N` is a 32-bit type (where the "T" multiplier doesn't fit).